tutorials/advanced_features/State Preparation with the SLM Mask.ipynb

```diff
@@ -19,7 +19,8 @@
    "metadata": {},
```

Comment:
<p>...if <code>bottom_detunig</code> is defined and <strong>its</strong> value is <strong>higher</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>